civis/ml/_model.py

```diff
@@ -631,7 +643,10 @@ def __init__(self, model, dependent_variable,
                  cross_validation_parameters=None, model_name=None,
                  calibration=None, excluded_columns=None, client=None,
                  cpu_requested=None, memory_requested=None,
-                 disk_requested=None, verbose=False):
+                 disk_requested=None, notification_urls=None,
+                 success_email_subject=None, success_email_body=None,
+                 success_email_addresses=None, failure_email_addresses=None,
+                 stall_warning_minutes=None, verbose=False):
```

Comment:
I slightly prefer just passing a notification dict so we're not maintain docs in two different places.  We'll also get around needing to default to "success_on" and "failure_on", which might be unexpected to the user.  If we were ever to expose `schedule`, too, I think we couldn't expose all the parameters.